src/core/RadioButton/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import useStyles from './styles';
+import { RadioBaseProps } from './base';
+import Radio from '@material-ui/core/Radio';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+interface RadioButtonProps extends RadioBaseProps {
+  children: string;
+  disabled?: boolean;
+  checked?: boolean;
+}
```

Comment:
This interface is not required since Radio has these types/props by default.

Reply:
@arkajyotiMukherjee Do you mean this method by saying ''This should extend Mui Radio button'' in of the comments?

Reply:
Radio API already provides these types, thus we don't require an interface unless we need some custom props. Here is the API for the same [https://material-ui.com/api/radio/](https://material-ui.com/api/radio/)